llvm/lib/CodeGen/MachineVerifier.cpp

```diff
@@ -1710,41 +1710,51 @@ void MachineVerifier::verifyPreISelGenericInstruction(const MachineInstr *MI) {
     }
 
     LLT DstTy = MRI->getType(MI->getOperand(0).getReg());
-    LLT Src0Ty = MRI->getType(Src0Op.getReg());
     LLT Src1Ty = MRI->getType(Src1Op.getReg());
 
     if (!DstTy.isVector()) {
       report("Destination type must be a vector", MI);
       break;
     }
 
-    if (!Src0Ty.isVector()) {
```

Comment:
Is it okay to remove this check?

Reply:
The relationship is already declared in the .td file and checked automatically. This check was redundant.